Makefile

```diff
@@ -1,19 +1,51 @@
 DOT := $(shell command -v dot 2> /dev/null)
 
-up:
+rm-up:
 	docker network inspect ssdcrmdockerdev_default >/dev/null || docker network create ssdcrmdockerdev_default
 	docker compose -f dev.yml -f rm-services.yml up -d ${SERVICE} ;
 	pipenv install --dev
 	./setup_pubsub.sh
+
+rh-up:
+	docker network inspect ssdcrmdockerdev_default >/dev/null || docker network create ssdcrmdockerdev_default
+	docker compose -f rh-dev.yml -f rh-services.yml up -d ${SERVICE} ;
+	pipenv install --dev
+	./setup_pubsub.sh
+
+rmrh-up:
+	docker network inspect ssdcrmdockerdev_default >/dev/null || docker network create ssdcrmdockerdev_default
+	docker compose -f dev.yml -f rm-services.yml -f rh-dev.yml -f rh-services.yml up -d ${SERVICE} ;
+	pipenv install --dev
+	./setup_pubsub.sh
 	
-down:
+rm-down:
+	@echo "IF RH IS RUNNING, THIS WILL STOP PUBUSB EMULATOR FOR THOSE SERVICES TOO"
 	docker compose -f dev.yml -f rm-services.yml down
 
-pull:
+rh-down:
+	@echo "IF RM IS RUNNING, THIS WILL STOP PUBUSB EMULATOR FOR THOSE SERVICES TOO"
+	docker compose -f rh-dev.yml -f rh-services.yml down
+
+rmrh-down:
+	docker compose -f dev.yml -f rm-services.yml -f rh-dev.yml -f rh-services.yml down
+
+rm-pull:
 	docker compose -f dev.yml -f rm-services.yml pull ${SERVICE}
 
-logs:
+rh-pull:
+	docker compose -f rh-dev.yml -f rh-services.yml pull ${SERVICE}
+
+rmrh-pull:
+	docker compose -f dev.yml -f rm-services.yml -f rh-dev.yml -f rh-services.yml pull ${SERVICE}
+
+rm-logs:
```

Comment:
I don't think we need separate, specific `logs` targets, we could just have one which looks across all our docker compose files